src/services/talkService.js

```diff
@@ -57,6 +61,66 @@ export async function createTalkRoom(eventTitle = null, eventDescription = null)
 	}
 }
 
+/**
+ *
+ * @param eventComponent
+ */
+export async function updateTalkParticipants(eventComponent) {
+	const apiVersion = loadState('calendar', 'talk_api_version')
+	const url = eventComponent.getConferenceList()[0]?.uri ?? eventComponent.location
+	if (!url || !url.startsWith(window.location.protocol + '//' + window.location.host)) {
+		logger.debug('Event\'s conference/location is from another host', url)
+		return
+	}
+	const token = url.match(/\/call\/([a-z0-9]*)$/)[1]
+	if (!token) {
+		logger.debug('URL ' + url + ' contains no call token')
+		return
+	}
+	try {
+		const participantsResponse = await HTTPClient.get(generateOcsUrl('apps/spreed/api/' + apiVersion + '/', 2) + 'room/' + token + '/participants')
+		// Ignore if the actor isn't owner of the conversation
+		if (!participantsResponse.data.ocs.data.some(participant => participant.actorId === getCurrentUser().uid && participant.participantType <= 2)) {
+			logger.debug('Current user is not a moderator or owner', { currentUser: getCurrentUser().uid, conversation: participantsResponse.data.ocs.data })
+			return
+		}
+
+		for (const attendee of eventComponent.getAttendeeIterator()) {
+			logger.debug('Processing attendee', { attendee })
+			if (['RESOURCE', 'ROOM'].includes(attendee.userType)) {
```

Comment:
**thought (non-blocking):** Not that I've ever encountered it, but you could exclude `'GROUP'` as well.